ote_sdk/ote_sdk/usecases/exportable_code/demo/demo.py

```diff
@@ -70,6 +70,20 @@ def build_argparser():
         default=False,
         action="store_true",
     )
+    args.add_argument(
```

Comment:
`--no_show` argument is added to CLI, but is not used in the code?

Reply:
It is passed to a [demo.run()](https://github.com/openvinotoolkit/training_extensions/blob/53afcc0cdf413b12037b84eccb01320ae75f23d9/ote_sdk/ote_sdk/usecases/exportable_code/demo/demo.py#L126) function below

`demo.run(args.input, args.loop and not args.no_show)`

Reply:
LGTM. I am surprised because the demo.run() line wasn't changed in this branch. Maybe that was a bug that is fixed in this PR.